app/models/annotation/AnnotationService.scala

```diff
@@ -496,21 +474,27 @@ class AnnotationService @Inject()(annotationInformationProvider: AnnotationInfor
           skeletonTracings <- getSkeletonTracings(dataSetId, annotations.map(_.skeletonTracingId))
           volumeTracings <- getVolumeTracings(dataSetId, annotations.map(_.volumeTracingId))
           volumeDataObjects <- getVolumeDataObjects(dataSetId, annotations.map(_.volumeTracingId))
```

Comment:
I decided to leave this as it was because we avoid sending unnecessary requests to the tracingstore